### ISSUE TITLE
Niche and Size Modules no longer running

### ISSUE THREAD

**Issue by ngotelli** [GotelliLab/EcoSimR#59]
Hi @emhart 

Parameters not being read in properly with `niche_null_model(dataMacWarb)` or with `size_null_model(dataRodents)`. Error occurs both in the console and in the `.Rmd` vignette. However, `cooc_null_model(dataWiFinches)` still runs OK.

I tried to push the vignette with this error, but generated a merge conflict. I will wait until this is resolved. :hand: :sleepy:

**Comment by emhart** [comment]
@ngotelli I don't get any error.  Can you post the error that you're getting?  However if you're getting a merge conflict, you'll have to resolve it before you can push the vignette.  What file is throwing the conflict?

**Comment by ngotelli** [comment]
Hmm, I was unable to replicate the error this morning, so I am not sure
what happened there.

    These merge conflicts have happened a few times before, and they may
have to do with the version number that is printed on dev tools or one of
the other libraries. I am still learning my way around git (and how to
resolve a merge conflict), so I have in those cases just recloned the repo
and started fresh.

    In the vignette .Rmd, is there any way to suppress the output of the
progress bar? I have tried the switches for echo, warnings, and messages
in the chunk header, but the progress bar still shows up, which we
definitely don't want to see in the rendered vignette (since it refreshes
20 or 30 times during a run).

    Thanks, and enjoy the weekend in sunny California (we are getting more
snow here).

    Nick

    Quoting Edmund Hart <notifications@github.com>:
>
> @ngotelli[1] I don't get any error. Can you post the error that  
> you're getting? However if you're getting a merge conflict, you'll  
> have to resolve it before you can push the vignette. What file is  
> throwing the conflict?
>
> —
>    Reply to this email directly or view it on GitHub[2].
>  
>
>

 
********************************************************
    Nicholas J. Gotelli       Office Phone: 802-656-0450
    Department of Biology     Lab Phone: 802-656-0451   
    University of Vermont     Fax: 802-656-2914
    Burlington, VT 05405      e-mail: ngotelli@uvm.edu
    ********************************************************
    Home Page (with manuscript pdfs):
http://www.uvm.edu/~ngotelli/homepage.html

    Musician's Corner (with mp3s):
http://www.uvm.edu/~ngotelli/musicpage/music.html

    NEW: EcoSimR (free software for null model analysis):
http://www.uvm.edu/~ngotelli/EcoSim/EcoSim.html
    ********************************************************



Links:
------
[1] https://github.com/ngotelli
[2] https://github.com/GotelliLab/EcoSimR/issues/59#issuecomment-78770949

**Comment by ngotelli** [comment]
Hi @emhart 

I pushed up the final version of the Niche Overlap Vignette. Thanks for adding the option to suppress the progress bar. That's pretty essential to be able to use the package in a markdown document! The only real problem I ran into was that `plot(type = "niche")` doesn't work in the vignette because the figure margins are too large (I have marked the line of code in the `.Rmd` file). Aaron and I ran into the problem a long time ago when we first started coding the graphics. We adjusted the margins so the niche plot does work with R studio, and I just checked that the plots in EcoSimR do work within a standard `.Rmd` file in RStudio. They just aren't working in the vignette template. 

For now I will move on and try to crank out a co-occurrence module next.

Best,

Nick
